### ISSUE TITLE
Ship lookup sometimes fails when only ship name is used

### ISSUE THREAD

**Issue by Carb0nNova** [galaban/LOTJShipStackerTracker#1]
As per Changes 1138, you have to use the whole ship name to look up ships.

Unfortunately, this doesn't prove effective enough to prevent you from looking up, say "Starfighter 'Bob and Joe" when you try to lookup "Freighter 'Bob'" if you just do "locateship bob"

My recommendation is that the following line be added to any lookup calls, namely LookupNextUnownedShip and LookupOldestShip

ship = row.type .. " '" .. row.name .. "'"
return ship

This will execute a locateship on Starfighter 'Bob and Joe', thus increasing the reliability of the command.

-@Domovoi

**Comment by galaban** [comment]
Good call.  Ship names aren't always reliable.  That's why I ended up putting in that deletenext command because of ships that had preceding values.  I have not tested the change, but it's in place now.
